src/javascript/app/pages/user/new_account/financial_acc_opening.js

```diff
@@ -21,6 +21,22 @@ const FinancialAccOpening = (() => {
             return;
         }
 
+        if (sessionStorage.getItem('is_risk_disclaimer')){
+            handleResponse(JSON.parse(sessionStorage.getItem('client_form_response')));
+        }
+
+        const setClientFormResponse = () => {
+            const client_form_response = sessionStorage.getItem('client_form_response') ? JSON.parse(sessionStorage.getItem('client_form_response')).echo_req : false;
+            if (!isEmptyObject(client_form_response)) {
+                const keys = Object.keys(client_form_response);
```

Comment:
do we need this declaration? it's only used once

Reply:
True, I previously called it twice, therefore created a function, but we don't need it anymore